### PR TITLE
Add a 2-level drilldown for token accounts

### PR DESCRIPTION
**Pull request by awrichar**
This depends on some API changes in https://github.com/hyperledger/firefly/pull/313.

FireFly_Explorer

FireFly_Explorer

FireFly_Explorer

### CODE REVIEW

**Comment by awrichar** [comment]
Changed the header on the account details page for a particular pool (just use the pool name). Here's a new screenshot, showing a fungible pool:

FireFly_Explorer

File: src/modules/tokens/views/Transfers/Transfers.tsx [hunk 1]
```diff
@@ -182,7 +182,7 @@ export const Transfers: () => JSX.Element = () => {
               address={tokenTransfer.to}
             />
           ) : (
-            '---'
+            '--'
```
Comment (eberger727) [inline src/modules/tokens/views/Transfers/Transfers.tsx:185]: should these be put in the translation file...just to remove hard coded strings?

File: src/modules/tokens/views/Accounts/AccountPools.tsx [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+  separator: {
+    flexGrow: 1,
+  },
+  bold: {
+    fontWeight: 'bold',
+  },
+  paddingBottom: {
+    paddingBottom: theme.spacing(2),
+  },
+  paddingRight: {
```
Comment (eberger727) [inline src/modules/tokens/views/Accounts/AccountPools.tsx:345]: unused in this file

File: src/modules/tokens/views/Accounts/AccountDetails.tsx [hunk 3]
```diff
@@ -246,9 +323,21 @@ export const AccountDetails: () => JSX.Element = () => {
...
+              underline="hover"
+              color="inherit"
+              sx={{ cursor: 'pointer' }}
+              onClick={() =>
+                history.push(
+                  `/namespace/${selectedNamespace}/tokens/accounts/${key}`
+                )
+              }
+            >
               {key}
```
Comment (eberger727) [inline src/modules/tokens/views/Accounts/AccountDetails.tsx:336]: do we want to use short keys here?

**Review by eberger727 — COMMENTED** [review]
Looks good, left some comments

File: src/modules/tokens/views/Transfers/Transfers.tsx (hunk 1 above)
Comment (awrichar) [reply src/modules/tokens/views/Transfers/Transfers.tsx:185]: Yea, probably a good idea

File: src/modules/tokens/views/Accounts/AccountDetails.tsx (hunk 3 above)
Comment (awrichar) [reply src/modules/tokens/views/Accounts/AccountDetails.tsx:336]: do we have a way to shorten the key without triggering a popover? the link here should just be to the previous page.

Comment (eberger727) [reply src/modules/tokens/views/Accounts/AccountDetails.tsx:336]: Probably can make a util function to shorten it

**Review by eberger727 — APPROVED** [review]

**Comment by awrichar** [comment]
Added a few additional changes to surface when a transfer has a message attached. Note that the "link" to the message is currently non-functional - might need a consult on how to link across modules to a specific message.

FireFly_Explorer

FireFly_Explorer

**Review by shorsher — APPROVED** [review]
Looks great, made this issue https://github.com/hyperledger/firefly-ui/issues/72 to track the work for creating a new message details view.

**Pull request merged**